packages/expo-router/plugin/options.json

```diff
@@ -136,6 +136,23 @@
             }
           }
         },
+        "headers": {
+          "description": "A list of header values that are sent with every response from the server",
```

Comment:
I think scoping this down is fine. As discussed, it's a bit unclear how to deal with "none route" responses unless we hoist this, the problem being:
- if we want this to apply in EAS Hosting to asset responses, we have to hoist this
- but, if we hoist this, we also then need to teach users how to apply this for every adapter, since adapters don't include assets (and instructions for Vercel/Netlify would differ and could get tricky)

So, overall, I'm fine with applying this to only routes that Expo Router controls, and treating assets as out-of-scope for now, (although, some hoisting could be applied there), but I'd change this description slightly to:

> A list of headers that are set on every route response from the server

To imply that it won't necessarily apply to assets.